src/commands/new-proj.ts

```diff
@@ -190,6 +207,92 @@ async function initQuarkusProject() {
   }
 }
 
+/**
+ * Attachs AWS to the project
+ * 
+ * @param {any} instance Vorpal instance for inquiry.
+ */
+async function attachAWS(instance: any) {
+  let access: string = null;
+  let secret: string = null;
+  try {
+    const configResult = await PromptUtils.listPrompt(
+      instance,
+      "Do you have an AWS config file under home location /.aws/config : ",
+      [ "Yes", "No" ]
+    );
```

Comment:
couldnt this be done with promptutils.confirmPrompt?

Reply:
Replaced with Promptutils.confirmPrompt